<p class="publicationTitle">A scalable platform for the development of cell-type-specific viral drivers</p>

<!-- Do not include the title in this citation, since that would be redundant with the title above. -->
<p class="publicationCitation">
	<!-- Make sure to include http:// in the link, otherwise it will be interpreted as a relative link. -->
	<a href="https://elifesciences.org/articles/48089">
	Hrvatin, Sinisa, Christopher P. Tzeng, M. Aurel Nagy, Hume Stroud, Charalampia Koutsioumpa, Oren F. Wilcox, Elena G. Assad, Jonathan Green, Christopher D. Harvey, Eric C. Griffith, and Michael E. Greenberg. (2019). eLife 8, pii: e48089.
	</a>
</p>

<!-- Case matters in the pdf file name. -->
<span class="buttonPdf" id="Hrvatin2019"><a href="pdf/hrvatin2019.pdf">Download PDF</a></span>

<div class="less">
	<span class="buttonShowMore" id="Hrvatin2019">Show abstract</span>
</div>

<!-- Make sure to replace special characters in the abstract with their HTML versions. -->
<div class="more">
	<p class="publicationAbstract">
	Enhancers are the primary DNA regulatory elements that confer cell type specificity of gene expression. Recent studies characterizing individual enhancers have revealed their potential to direct heterologous gene expression in a highly cell-type-specific manner. However, it has not yet been possible to systematically identify and test the function of enhancers for each of the many cell types in an organism. We have developed PESCA, a scalable and generalizable method that leverages ATAC- and single-cell RNA-sequencing protocols, to characterize cell-type-specific enhancers that should enable genetic access and perturbation of gene function across mammalian cell types. Focusing on the highly heterogeneous mammalian cerebral cortex, we apply PESCA to find enhancers and generate viral reagents capable of accessing and manipulating a subset of somatostatin-expressing cortical interneurons with high specificity. This study demonstrates the utility of this platform for developing new cell-type-specific viral reagents, with significant implications for both basic and translational research.<span class="buttonShowLess" id="Hrvatin2019">Hide abstract</span>
	</p>
</div>